vendor/bat-native-ledger/src/bat/ledger/internal/gemini/gemini_util.cc

```diff
@@ -114,7 +108,7 @@ type::ExternalWalletPtr GenerateLinks(type::ExternalWalletPtr wallet) {
 
   const std::string auth_url = GetAuthorizeUrl(wallet->one_time_string);
 
-  wallet->verify_url = GetVerifyUrl();
```

Comment:
Did the url returned by `GetVerifyUrl` used to be required? Or just not needed after testing things?

Reply:
`verifyURL` was used for the connected state, we removed the state and show an error if the user has a gemini account but not verified.